### PR TITLE
Codenvy 248

### PR DESCRIPTION
**Pull request by mshaposhnik**

### CODE REVIEW

File: core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    private final MachineTokenRegistry registry;
+
+    @Inject
+    public  MachineTokenService(MachineTokenRegistry machineTokenRegistry) {
+        this.registry = machineTokenRegistry;
+    }
+
+
+    @GET
+    @Produces(APPLICATION_JSON)
```
Comment (mshaposhnik) [inline core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java]: Do we need @RolesAllowed("user") here ?

File: core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+/**
+ * Machine token service.
+ *
+ * @author Max Shaposhnik (mshaposhnik@codenvy.com)
+ */
+
+@Api(value = "/machine/token", description = "Machine token REST API")
+@Path("/machine/token")
+public class MachineTokenService {
```
Comment (skabashnyuk) [inline core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java]: codenvy-hosted-platform-api-impl move it here

File: core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java (hunk 1 above)
Comment (skabashnyuk) [inline core/codenvy-hosted-machine-token-service/src/main/java/com/codenvy/service/machine/token/MachineTokenService.java]: yes

File: core/codenvy-hosted-machine-token-service/pom.xml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>codenvy-hosted-infrastructure-parent</artifactId>
+        <groupId>com.codenvy.hosted-infrastructure</groupId>
+        <version>4.1.0-RC1-SNAPSHOT</version>
+    </parent>
+    <artifactId>codenvy-hosted-machine-token-service</artifactId>
+    <properties>
+        <dto-generator-out-directory>${project.build.directory}/generated-sources/dto/</dto-generator-out-directory>
+        <findbugs.failonerror>false</findbugs.failonerror>
```
Comment (sleshchenko) [inline core/codenvy-hosted-machine-token-service/pom.xml]: Can we remove this line?

File: core/codenvy-hosted-machine-token-service/pom.xml [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <dtoPackages>
+                                <package>com.codenvy.service.token.shared.dto</package>
+                            </dtoPackages>
+                            <outputDirectory>${dto-generator-out-directory}</outputDirectory>
+                            <genClassName>com.codenvy.service.machine.token.shared.dto.server.DtoServerImpls</genClassName>
```
Comment (sleshchenko) [inline core/codenvy-hosted-machine-token-service/pom.xml]: com.codenvy.service.token.server.dto.DtoServerImpls or com.codenvy.service.token.shared.dto.DtoServerImpls?

**Pull request closed without merging**